### PR TITLE
added potatoe salad recipe

### PR DESCRIPTION
**Pull request by LukaPlavec**
added detailed instructions

### CODE REVIEW

File: desserts/potato_salad.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Potato salad
+
+## Ingredients
+- **1 kg** potatos
```
Comment (luisdavidpabonospina) [inline desserts/potato_salad.md:4]:
```suggestion
- **1 kg** potatoes
```

Comment (Ali-Alshawabkeh) [inline desserts/potato_salad.md:4]:
```suggestion
- **1 kg** potatos
```

File: desserts/potato_salad.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- **1 kg** spaghetti
```
Comment (luisdavidpabonospina) [inline desserts/potato_salad.md:5]: spaghetti in a potato salad?

File: desserts/potato_salad.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- **0.1 kg** onions
+- **500 ml** broth
+
+
+
+## Instructions
+
+1. Boil the potatoes
+
+2. Cut the potatoes in pieces
```
Comment (luisdavidpabonospina) [inline desserts/potato_salad.md:15]:
```suggestion
2. Cut the potatoes into pieces
```

File: desserts/potato_salad.md (hunk 2 above)
Comment (LukaPlavec) [reply desserts/potato_salad.md:5]: You're right, that's a typo. Will change ;)

**Pull request merged**